tests/foreman/api/test_syncplan.py

```diff
@@ -0,0 +1,399 @@
+"""Unit tests for the ``repositories`` paths."""
+import datetime
```

Comment:
we could just import datetime and timedelta from datetime here, if you want to cut down on namespace resolution and import bulk. not even close to a blocker!